docs/guides/unity-catalog.md

```diff
@@ -1,12 +1,132 @@
 ---
-page_title: "Unity Catalog setup on AWS"
+page_title: "Unity Catalog set up on AWS"
 ---
 
+# Deploying pre-requisite resources and enabling Unity Catalog (AWS Preview)
+
+-> **Private Preview** This feature is in [Private Preview](https://docs.databricks.com/release-notes/release-types.html). Contact your Databricks representative to request access. 
+
+Databricks Unity Catalog brings fine-grained governance and security to lakehouse data using a familiar, open interface. Deploying the underlying cloud resources and Unity Catalog objects can be done with Terraform to simplify the set up and enable automation
+
+This guide assumes you have `databricks_account_username` and `databricks_account_password` for [https://accounts.cloud.databricks.com](https://accounts.cloud.databricks.com) and can find `databricks_account_id` in the bottom left corner of the page, once you're logged in. This guide is provided as-is and assumes you'll use it as the basis for your setup.
+
+Below are the high level steps to get started with Unity Catalog:
```

Comment:
probably add something like `and you can use this guide as the basis for your custom Terraform module`

Reply:
done